Makefile

```diff
@@ -244,7 +244,7 @@ start-e2e:
 
 .PHONY: test-e2e
 test-e2e:
-	go test -timeout 30m -v -count 1 --tags e2e -p ${E2E_PARALLEL} --short ./test/e2e ${E2E_TEST_OPTIONS}
+	go test -timeout 60m -v -count 1 --tags e2e -p ${E2E_PARALLEL} --short ./test/e2e ${E2E_TEST_OPTIONS}
```

Comment:
Can you revert this change? The e2e failure was not related to exceeding 30m timeout, but generally flakiness, which we are working on.

Reply:
Hey @jessesuen! Sure! I saw a couple of actions that were timing around 30m so figured I'd try running the action with an increased timeout. Reverting the change immediately.